integration-tests/slack/slack-notify-channels.json

```diff
@@ -0,0 +1,157 @@
+{
+  "pass": [
+    {
+      "name": "dsde-qa",
+      "id": "C53JYBV9A"
+    }
+  ],
+  "fail": [
+    {
+      "name": "dsde-qa",
+      "id": "C53JYBV9A",
```

Comment:
Where do these IDs come from?

Reply:
`channel id`. It can be found in Slack channel modal.

<img width="427" alt="Screen Shot 2022-07-26 at 2 01 34 PM" src="https://user-images.githubusercontent.com/35533885/181078944-f3ff27ff-90d6-449f-b51b-8556a2d9a148.png">